Identity Backend Audit HMAC Request/Response Cannot Be Configured

**Describe the bug**

It does not appear possible to allowlist fields to be printed in cleartext in the audit logs for requests/responses from the Identity backend.

Based on the Identity backend docs and poking around at a local Vault dev server, it appears that the `tune` option used to allow fields to be printed in plaintext rather than HMAC'd values in audit logs in secret/auth backends is not available for the Identity backend.

It would be useful to our operations and use of the Identity backend if we were able to allow specific fields (ex. `policies` or `member_entity_ids`) from the request or response of Identity backend request to be printed in plaintext in our audit logs 

**To Reproduce**
Steps to reproduce the behavior:
Run `vault server -dev` and configure VAULT_ADDR/VAULT_TOKEN to work with that:
```
> vault secrets tune -audit-non-hmac-request-keys=member_entity_ids -audit-non-hmac-request-keys=member_entity_ids identity
Error tuning secrets engine identity/: Error making API request.

URL: POST http://127.0.0.1:8200/v1/sys/mounts/identity/tune
Code: 400. Errors:

* cannot tune "identity/"

> vault write sys/mounts/identity/tune audit-non-hmac-request-keys=member_entity_ids
Error writing data to sys/mounts/identity/tune: Error making API request.

URL: PUT http://127.0.0.1:8200/v1/sys/mounts/identity/tune
Code: 400. Errors:

* cannot tune "identity/"
```

On the server logs, running the above client commands will result in:
```
2022-03-24T11:25:17.201-0400 [ERROR] secrets.system.system_fafb1c8d: cannot tune this mount: path=identity/
```

**Expected behavior**

The Identity mount can be tuned to allow specific fields to be printed in audit logs in plaintext

**Environment:**
* Vault Server Version (retrieve with `vault status`): v1.9.3
* Vault CLI Version (retrieve with `vault version`): v1.9.3
* Server Operating System/Architecture: OS X, Linux

Vault server configuration file(s): n/a, reproduces on a `vault server -dev` instance

**Additional context**
None

--- a/changelog/14723.txt
+++ b/changelog/14723.txt
@@ -1,0 +1,4 @@
+```release-note:improvement
+core/identity: allow identity backend to be tuned using standard secrets backend tuning parameters.
+```
+

--- a/vault/mount.go
+++ b/vault/mount.go
@@ -110,7 +110,6 @@ var (
 		mountPathCubbyhole,
 		mountPathSystem,
 		"audit/",
-		mountPathIdentity,
 	}
 
 	// singletonMounts can only exist in one location and are

